pkg/controller/jobframework/integrationmanager.go

```diff
@@ -136,6 +137,36 @@ func RegisterIntegration(name string, cb IntegrationCallbacks) error {
 	return manager.register(name, cb)
 }
 
+func RegisterExternalIntegration(name string) error {
+	splitName := strings.Split(name, ", Kind=")
+	if len(splitName) != 2 {
+		return fmt.Errorf("%v does not match expected format '$GROUP/$VERSION, Kind=$KIND'", name)
+	}
+	apiVersion := splitName[0]
+	kind := splitName[1]
+	obj := &metav1.PartialObjectMetadata{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: apiVersion,
+			Kind:       kind,
+		},
+	}
+	cb := IntegrationCallbacks{
+		// JobType and IsManagingObjectsOwner are relevant
```

Comment:
These are a lot of things we don't need. Maybe just add a dedicated list into the manager for external kinds.